### PR TITLE
Documentation/Manifests: Improve YMLs.

### PR DESCRIPTION
**Pull request by Gacko**

### CODE REVIEW

**Comment by tomdee** [comment]
Could you make this change to the other two manifest files in this repo too?

**Comment by Gacko** [comment]
Done. I did not change the legacy init containers in some manifests since it would make them hardly readable.

File: Documentation/kube-flannel-aliyun.yml [hunk 1]
```diff
@@ -93,7 +94,7 @@ spec:
       serviceAccountName: flannel
       initContainers:
       - name: install-cni
-        image: registry.cn-hangzhou.aliyuncs.com/google-containers/flannel:v0.9.0
+        image: registry.cn-hangzhou.aliyuncs.com/google-containers/flannel:v0.9.1
```
Comment (tomdee) [inline Documentation/kube-flannel-aliyun.yml]: Does this image actually exist? I don't have permissions to push to this repo...

**Comment by tomdee** [comment]
@Gacko Looks good, but I'm not sure about bumping the aliyun image - it might be safer to just leave that at 0.9.0

Can you squash this commits into one and I'll merge.

File: Documentation/kube-flannel-aliyun.yml (hunk 1 above)
Comment (Gacko) [reply Documentation/kube-flannel-aliyun.yml]: Me neither, I thought you already did so. I'll revert this change.

**Comment by Gacko** [comment]
I removed the commit changing the version of the image from the aliyun registry.

**Pull request merged**